shacl12-overview/index.html

```diff
@@ -346,30 +346,118 @@
 			<p>
-				This specification is published by the <a href="https://www.w3.org/groups/wg/data-shapes/">Data Shapes Working Group</a>.
+				The <em>Shapes Constraint Language</em>, "SHACL", is a language for validating RDF graphs against a set
+				of conditions. SHACL has a core specification, extensions for related query languages, User Interface
+				generation, rule-based reasoning and a data profiling mechanism. This document overviews all these
+				parts of SHACL and serves as an introduction to all of them and related on-going work such as libraries
+				of examples.
+			</p>
+			<p>
+				This documentation is published by the <a href="https://www.w3.org/groups/wg/data-shapes/">Data Shapes Working Group</a>.
 			</p>
 		</section>
 
 		<section id="sotd"></section>
 
 		<section id="introduction">
 			<h2>Introduction</h2>
-			<p class="todo">TODO</p>
+			<p>
+				This document provides a non-normative, high-level, overview of the various Shapes Constraint Language (SHACL) specifications.
+			</p>
+			<p>
+				The second iteration of the World Wide Web Consortium's <a href="https://www.w3.org/groups/wg/data-shapes/">Data Shapes Working Group</a> aimed to "...update data shapes standards in line with the versions of core Semantic Web standards that cater for RDF-star and to extend the applications of data shapes with new packaging and use specifications". The outcome of the Working Group's operations was a set of specifications - W3C 'Recommendations' - that cover core SHACL elements, extended ones and SHACL functions beyond RDF validation, such as User Interface generation.
+			</p>
+			<p>
+				The <em>What's new in SHACL 1.2</em> section below describes the changes between SHACL, as published in 2017 and SHACL 1.2, including all the new SHACL elements such as SHACL for User Interface generation.
+			</p>
+			<p>
+				The <em>Specifications</em> section below lists all the Data Shapes Working Group's specifications which are also listed in each specification's Introduction.
+			</p>
+			<p>
+				The appendixes below contain material supporting the set of SHACL specifications and provide SHACL users with additional support, such as validators for shapes graphs.
+			</p>
 		</section>
 
-		<section id="shapes">
-			<h2>Shapes</h2>
-			<p class="todo">TODO: This section will describe the Shapes created for SHACL-SHACL.</p>
+		<section id="whatsnew">
+			<h2>What's New in SHACL 1.2</h2>
 			<p>
-				These shapes are available in the RDF resource at <a href="http://www.w3.org/ns/shacl-shacl">http://www.w3.org/ns/shacl-shacl</a>.
+				<em>coming soon</em>
 			</p>
 		</section>
 
-		<section id="governance">
-			<h2>Governance</h2>
-			<p class="todo">TODO: This section will describe the governance arrangements in place for the continued maintenance of this NOTE.</p>
+		<section id="specifications">
+			<h2>Specifications</h2>
+			<p>
+				SHACL 1.2:
+			</p>
+			<ol>
```

Comment:
@nicholascar Could I interest you in the `dl`/`dt`/`dd` element-set?  (Fine if no, I won't fight for it.)

Reply:
Sure! I just copied HTML from RDF 1.2, but indeed W3C often uses the `dl` set, so happy to change to that.

Done now.